tests/e2e/support/objects/app-files/link/index.ts

```diff
@@ -79,4 +81,17 @@ export class Link {
     })
     await this.#page.goto(startUrl)
   }
+
+  async publicLinkVisible(linkName): Promise<string[]> {
```

Comment:
publicLinkVisible naming is misleading to me, can you tell what you want to achieve

Reply:
It is to get the URL of the public link.

Reply:
It's now changed to `getPublicLinkUrl`